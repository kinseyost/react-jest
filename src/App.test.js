import React from 'react';
import { shallow } from 'enzyme';
import App from './App';
import { shallowToJson } from 'enzyme-to-json'

test('renders without crashing', () => {
  const wrapper = shallow(<App />);
  const tree = shallowToJson(wrapper);
  expect(tree).toMatchSnapshot();
});
