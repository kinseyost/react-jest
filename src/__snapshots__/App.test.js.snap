exports[`test renders without crashing 1`] = `
<div
  className="App">
  <div
    className="App-header">
    <h2>
      Welcome three React
    </h2>
  </div>
  <p
    className="App-intro">
    To get started, edit 
    <code>
      src/App.js
    </code>
     and save to reload.
  </p>
</div>
`;
